src/Hocon/Impl/HoconValue.cs

```diff
@@ -681,7 +681,7 @@ internal void ReParent(HoconValue value)
         /// <returns>A HOCON string representation of this <see cref="HoconValue"/>.</returns>
         public override string ToString()
         {
-            return ToString(0, 2);
+            return ToString(1, 2);
```

Comment:
Seems like someone changed `ToString` implementation during debugging something and did not update this `ToString` overload.